app/src/main/java/com/zulip/android/LoginActivity.java

```diff
@@ -198,6 +198,7 @@ public void onTaskFailure(String result) {
                     // Invalidate the token and try again, unless the user we
                     // are authenticating as is not registered or is disabled.
                     if (!loginTask.userDefinitelyInvalid) {
+                        GoogleAuthUtil.invalidateToken(that, token);
```

Comment:
Is this commit supposed to be here?  Commit message doesn't totally match the commit.